README.md

```diff
@@ -165,23 +168,77 @@ In Device Simulator Express, you can use keyboard to interact with the device:
   - Acceleration detection
   - Temperature sensor
 
+### Useful Links
+- Tutorials and Example Code for BBC micro:bit:
+  - [MicroPython documentation](https://microbit-micropython.readthedocs.io/en/latest/)
+  - [BBC micro:bit examples on the official micro:bit website](https://microbit.org/projects/make-it-code-it/?filters=python)
+
 ### How to use
+
+#### 1. Start with the "Device Simulator Express [micro:bit]: New File" Command.
+
+1. Type in `"Device Simulator Express: [micro:bit] New File"` in the command palette (`CTRL/CMD + SHIFT + P` to open the command palette).  
+   <img alt='"New File" animation' src='assets/readmeFiles/microbit/microbit-new-file.gif'>
+2. Name and save your file somewhere, and we’re good to go!
+
+#### 2. Start from an existing python file.
+
+1. Open the folder or your .py file in Visual Studio Code.
+2. Run `Device Simulator Express: [micro:bit] Open Simulator` from the command palette or icon in the editor toolbar.
+
+#### 3. Run your code on the simulator.
+
+<img alt='How to run the simulator animation' src='assets/readmeFiles/microbit/microbit-run.gif'>
+
+- Run `Run Simulator` from the command palette or use the `Play` button on the simulator webview.
+
+#### 4. Deploy your code to the physical device
+
+- Run `[micro:bit] Deploy to Device` from the command palette
+
+<img alt="Deploy to Device" src="assets/readmeFiles/microbit/microbit-deploy.png">
+
+#### 5. Use the Serial Monitor for your BBC micro:bit device (available on Windows and Mac only)
+
+1. Plug in your BBC micro:bit.
+2. Run the command `"Device Simulator Express: Open Serial Monitor"`.
+3. Select your baud rate for the serial port.
+4. The print() statements in your code will show in the output console.

```

Comment:
_&ast;nit&ast;_
Not a big fan of lists within lists that use same symbols to order the elements (E.g. "5. Use the serial monitor" followed by "1. Plug in your BBC micro:bit"). Perhaps consider using different symbols for the inner lists (E.g. ①, ②, ③;i), ii), iii), iv); etc.)? Or maybe something like 1.1, 1.2, 2.1, 3.1, 3.2, 3.3.
Also steps 3 (Run your code on the simulator) and 4 (Deploy your code to the physical device) have single "sub-step" formatted as unordered list, while all other steps have their "sub-steps" as ordered lists.
I think not changing anything is fine, just wanted to point these things out.

Reply:
I agree. Very very good points. I'll update it to something with that in mind.